wiring/inc/spark_wiring_platform.h

```diff
@@ -166,5 +170,9 @@
 #define Wiring_Watchdog 0
 #endif // Wiring_Watchdog
 
+#ifndef Wiring_Ledger
+#define Wiring_Ledger 0
+#endif
+
```

Comment:
Please add Variant and Map defines here as well in case we want to reuse them for external libraries.
```cpp
#define Wiring_Variant 1
#define Wiring_Map 1
```